src/index.js

```diff
@@ -78,7 +80,7 @@ const VLazyImageComponent = {
     }
   },
   destroyed() {
-    if ("IntersectionObserver" in window) {
+    if (this.observer) {
       this.observer.disconnect();
     }
```

Comment:
This issue I could not reproduce, only got error reports through sentry.io. They are I guess specific to some browsers or maybe caused by rat races.

Sometimes `this.observer` is `null`, so the call fails. Maybe it's when component is destroyed before it was mounted, or maybe it has to do with lazy hydration. Can't tell. But this fix helps.